tfhe/src/core_crypto/algorithms/test/modulus_switch_compression.rs

```diff
@@ -1,5 +1,4 @@
 use super::*;
```

Comment:
we need the equivalent for the multi bit right ?

Reply:
The decompression does not return a ciphertext
We must go through a multi bit BR here, which is tested in a doctest

Reply:
fair enough but I would like a core test allowing to test the multi bit ms compression/decompression that is a bit more thorough than a doctest

Also taking coefficients sufficiently spaced you should be able to check the decryption nonetheless for the multi bit CT right ? as you store each mask mod switched ?

given you already have the doctest, adapting should be ok I would think ?